src/main/msp/msp.c

```diff
@@ -3138,28 +3293,22 @@ static mspResult_e mspProcessInCommand(mspDescriptor_t srcDesc, int16_t cmdMSP,
 #ifdef USE_SIMPLIFIED_TUNING
     // Added in MSP API 1.44
     case MSP_SET_SIMPLIFIED_TUNING:
-        currentPidProfile->simplified_pids_mode = sbufReadU8(src);
-        currentPidProfile->simplified_master_multiplier = sbufReadU8(src);
-        currentPidProfile->simplified_roll_pitch_ratio = sbufReadU8(src);
-        currentPidProfile->simplified_i_gain = sbufReadU8(src);
-        currentPidProfile->simplified_d_gain = sbufReadU8(src);
-        currentPidProfile->simplified_pi_gain = sbufReadU8(src);
-#ifdef USE_D_MIN
-        currentPidProfile->simplified_dmin_ratio = sbufReadU8(src);
-#else
-        sbufReadU8(src);
-#endif
-        currentPidProfile->simplified_feedforward_gain = sbufReadU8(src);
-        currentPidProfile->simplified_pitch_pi_gain = sbufReadU8(src);
-
-        currentPidProfile->simplified_dterm_filter = sbufReadU8(src);
-        currentPidProfile->simplified_dterm_filter_multiplier = sbufReadU8(src);
-
-        gyroConfigMutable()->simplified_gyro_filter = sbufReadU8(src);
-        gyroConfigMutable()->simplified_gyro_filter_multiplier = sbufReadU8(src);
-
-        applySimplifiedTuning(currentPidProfile);
-
+        readSimplifiedPids(currentPidProfile, src);
+        readSimplifiedDtermFilters(currentPidProfile, src);
+        readSimplifiedGyroFilters(gyroConfigMutable(), src);
+        applySimplifiedTuning(currentPidProfile, gyroConfigMutable());
+        break;
```

Comment:
What if we were to add a new pid slider? It would have to be added to the end of this message and not in ```readSimplifiedPids```. Have a look at ```MSP_PID_ADVANCED```/```MSP_SET_PID_ADVANCED``` to see how we're dealing with removal and adding of new parameters.

Reply:
@klutvott123 good call.
So should I add like 5-10 dummy read/writes to accommodate future addings?

Reply:
done

Reply:
Having made my comment about reserving spare bytes for future expansion I come to this exchange! I'm still inclined to say we should only add the code required to support a new feature when we implement that feature to help keep code size under control.

Reply:
This message should really mirror what's going on in ```MSP_SIMPLIFIED_TUNING```. The approach with wrapping this in functions will work when there's just one of them as any new parameter could be added within the function, but here a new pid slider would have to be added AFTER ```readSimplifiedTuning``` and not in ```readSimplifiedPids```.